databroker/src/broker.rs

```diff
@@ -1622,12 +1630,14 @@ impl<'a, 'b> AuthorizedAccess<'a, 'b> {
     pub async fn subscribe(
         &self,
         valid_entries: HashMap<i32, HashSet<Field>>,
-    ) -> Result<impl Stream<Item = EntryUpdates>, SubscriptionError> {
+        lag_buffer_capacity: Option<usize>,
+    ) -> Result<impl Stream<Item = Result<EntryUpdates, BroadcastStreamRecvError>>, SubscriptionError>
+    {
         if valid_entries.is_empty() {
             return Err(SubscriptionError::InvalidInput);
         }
 
-        let (sender, receiver) = mpsc::channel(10);
+        let (sender, receiver) = broadcast::channel(lag_buffer_capacity.unwrap_or(1));
```

Comment:
I don't think we should have the ability to set an unbounded (or 2^32) capacity here, as this is controlled by third party apps. That would probably be an easy denial of service vector.

Reply:
Good point. We do not yet have any config framework for Databroker where one can set limits like this, right? so for now just hard-code a sane max-limit (1000?) and give an error if exceeded.

(And give a hint in proto file, like "max value supported is implementation dependent")

Reply:
Done